source/guides/core-concepts/cypress-studio.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Cypress Studio
+---
+
+{% note info %}
+# {% fa fa-graduation-cap %} What you'll learn
+
+- How to extend tests interactively using the Cypress Studio
+{% endnote %}
+
+# Overview
+
+Cypress Studio provides a visual way to generate tests within the Test Runner, by *recording interactions* against the application under test.
+
+{% imgTag /img/guides/cypress-studio/cypress-studio-overview.png "Cypress Studio" "no-border" %}
+
+The {% url `.click()` click %}, {% url `.dblclick()` dblclick %}, {% url `.type()` type %} and {% url `.select()` select %} Cypress commands are supported and will generate test code when interacting with the DOM inside of the Cypress Studio.
+
+## Using Cypress Studio
+
+The Cypress {% fa fa-github %} {% url "Real World App (RWA)" https://github.com/cypress-io/cypress-realworld-app %} is an open source project implementing a payment application to demonstrate real-world usage of Cypress testing methods, patterns, and workflows. It will be used to demonstrate the functionality of Cypress Studio.
+
+### Extending a Test
+
+You can extend any preexisting test or start by creating a new test under `cypress/integration` with the following test scaffolding.
+
+```js
+describe('Cypress Studio Demo', function () {
+  beforeEach(function () {
+    // Seed database with test data
+    cy.task('db:seed')
+
+    // Login test user
+    cy.database('find', 'users').then((user) => {
+      cy.login(user.username, 's3cret', true)
+    })
```

Comment:
I would discourage against having code here that can't be copy/pasted and work with no other code in their project. People **will** copy/paste this and get an error like 'cy.database is not defined' and 'cy.login is not defined' and be confused.